libvast/vast/schema.hpp

```diff
@@ -79,4 +79,6 @@ class schema : detail::equality_comparable<schema> {
 
 bool convert(const schema& s, json& j);
 
+caf::expected<schema> load_schema_file(const path& sf);
```

Comment:
You could consider to name this function `load_schema` with two overloads, one with a `const path& file` argument and the other with `const vector<path>& dirs`.